packaging/nfpm/nfpm.sh

```diff
@@ -30,14 +32,15 @@ for name in metaconvert mimir-continuous-test mimir mimirtool query-tee ; do
             # Generate package dependencies using envsubst
             mkdir "${pkg_dependencies_path}"
 
-            if [ -d "packaging/nfpm/${name}" ]; then
-              for dependencie in $(ls packaging/nfpm/${name}); do
```

Comment:
Does shellcheck complain about quoting the whole string (`"packaging/nfpm/${name}"`), or is it just the `ls` thing it complains about? Otherwise, I don't see the problem with quoting the whole string as before.

Reply:
It complained about iterating over the output of "ls". 
The change to quoting only the variable was an opinionated one (and both are valid).

Reply:
Yeah I know both are valid, I think our opinions just differ on what's preferable :)

Reply:
Oh, I probably don't have any preference. I just refactored multiple times this before committing, and ended up writing it _this way_ instead of _that way_.